src/main/resources/templates/fragments/inputs/textArea.html

```diff
@@ -6,17 +6,24 @@
       hasAriaLabel=${!#strings.isEmpty(ariaLabel)},
       rows=${!#strings.isEmpty(rows)} ? ${rows} : '6',
       maxLength=${!#strings.isEmpty(maxLength)} ? ${maxLength} : '500',
+      isRequiredInput=${requiredInputs.getOrDefault(inputName, false) || required != null && required},
```

Comment:
I would add parenthesis around the `required` part, like so `(required != null && required)`.

Reply:
Done